projects/plugins/automattic-for-agencies-client/tests/e2e/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+	"extends": "_jetpack-e2e-commons/tsconfig.json",
+	"compilerOptions": {
+		"baseUrl": ".",
+		"paths": {
+			"@e2e-commons/*": [ "node_modules/_jetpack-e2e-commons/*" ],
+			"@logger": [ "node_modules/_jetpack-e2e-commons/logger" ],
+			"@fixtures/*": [ "node_modules/_jetpack-e2e-commons/fixtures/*" ],
+			"@utils/*": [ "node_modules/_jetpack-e2e-commons/utils/*" ]
```

Comment:
I don't think this is a good idea or even worth it. These can easily be misunderstood as being some local modules, instead of being from a consumed package.

Instead of this, we may simply want to rename the `_jetpack-e2e-commons` package name to something simpler - `_jp-e2e`

Reply:
Thanks, I was not convinced myself. I think I revert these.

Reply:
Done, removed the alias paths. Can you review again @manzoorwanijk? Thanks